src/main/java/juke/Storage.java

```diff
@@ -0,0 +1,105 @@
+package juke;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class Storage {
```

Comment:
No javadoc for class? Maybe a short description of what the class does might be helpful!